### DISCUSSION TITLE
Can I embed a view of my workflow in another application?

### DISCUSSION THREAD

**Discussion by justinmchase** [argoproj/argo-workflows#5762] [Q&A] [answered]
I would love to have a simplified read-only view of a particular workflow so that I can embed it into my application for non-technical users to see.

I'm wondering if there are any UI components that I could add to my site which would know how to render argo workflow models but leave it up to me to figure out how to retrieve the data?

My thought is to create a high level workflow which is more business oriented but then have some / all steps themselves be sub-workflows which may have more complexity in them. If I could render the workflows for business it could help them get a broad understanding of the flow and more effectively monitor the state of the system. I don't necessarily want them to log into the argo UI itself or manage the workflows outside of their business-centric UI, but to be able to see them would be very useful.

**Comment by alexec** [comment, marked as answer]
Yes. Search the documents for "widgets"

**Reply by justinmchase** [reply]
Got it here it is:
https://argoproj.github.io/argo-workflows/widgets/

Documentation is a little anemic but it's good to know it's possible. I'll look into it further.
